app/config/production.json

```diff
@@ -0,0 +1,4 @@
+{
+  "OPENSHIFT_APP_NAMESPACE": "ff61fb-prod",
```

Comment:
Don't need to define this as it will be overridden by an env variable

Reply:
Removed